File: output/llama3.1-8b/avatar/Python/Java/atcoder_ABC139_B.java
import java.util.Scanner;

public class atcoder_ABC139_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int a = Integer.parseInt(input[0]);
        int b = Integer.parseInt(input[1]);

        int hole = 1;
        int i = 0;
        boolean check = true;

        while (check) {
            if (hole >= b) {
                check = false;
            } else {
                hole = a * (i + 1) - i;
                i++;
            }
        }

        System.out.println(i);
    }
}

File: output/llama3.1-8b/avatar/Java/Python/atcoder_ABC139_B.py
import sys
sc = input()
A = int(sc.split()[0])
B = int(sc.split()[1])
amari = 0
result = 0
result = (B - 1) // (A - 1)
amari = (B - 1) % (A - 1)
if amari != 0:
    result += 1
print(result)